test/test-server/org_handlers.go

```diff
@@ -57,17 +60,51 @@ func handleOrgEnvironments(t *testing.T) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 		if r.Method == http.MethodGet {
-			environmentList := &orgv2.OrgV2EnvironmentList{Data: getOrgEnvironmentsList(OrgEnvironments)}
+			environmentList := &orgv2.OrgV2EnvironmentList{Data: getOrgResourcesList[orgv2.OrgV2Environment](OrgEnvironments)}
 			err := json.NewEncoder(w).Encode(environmentList)
 			require.NoError(t, err)
 		}
 	}
 }
 
-func getOrgEnvironmentsList(envs []*orgv2.OrgV2Environment) []orgv2.OrgV2Environment {
-	envList := []orgv2.OrgV2Environment{}
-	for _, env := range envs {
-		envList = append(envList, *env)
+// Handler for: "/org/v2/organizations/{id}"
+func handleOrgOrganization(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		orgId := vars["id"]
+		w.Header().Set("Content-Type", "application/json")
+
+		if r.Method == http.MethodGet {
```

Comment:
It would be great to get into the habit of using `switch` statements here. It results in less redundancy when there are multiple methods.